jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/SelectedLogGroup.kt

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights
+
+import java.time.Instant
+import java.util.Date
+
+data class SelectedLogGroups(
+    var logGroups: String? = null
```

Comment:
This should also be a list? (at least add a TODO)

Reply:
Added a TODO